### ISSUE TITLE
Convert Custom Yolov4 model

### ISSUE THREAD

**Issue by sharoseali** [taipingeric/yolo-v4-tf.keras#1]
HI! I want to ask that I want to convert Yolov4 which has trained on 12 classes, hence I change some filter values. Will this repo will change my weights to the h5 Keras model successfully ??

**Comment by taipingeric** [comment]
Yes, remember to set the right class names file to build your model. You can check if you got a error message from [here](https://github.com/taipingeric/yolo-v4-tf.keras/blob/f580742b6aca675c8ba1e9e3cc4178a0cd2f6d71/utils.py#L50)
